Automattic/wp-calypso
# 100 year plan: Better animated Loader

Related to #

## Proposed Changes

* Current the loading screens for the 100 year flow are images. This change adds an animated rotating loader to the processing screens in the flow and right before the thank you page.

## Testing Instructions
- Go to /setup/hundred-year-plan/ 
- Continue through the flow
- After the domain step, make sure the loader is  as shown below
- If you want view the loader step for a bit longer comment out the line below
https://github.com/Automattic/wp-calypso/blob/f172acab760f688be9a032f458c9872ad1150a92/client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/index.tsx#L103
- Then after purchase make sure right before the thank you page the loader is as shown

https://github.com/Automattic/wp-calypso/assets/3422709/7d7c96f7-8169-4412-afcb-8f815a995d9d

https://github.com/Automattic/wp-calypso/assets/3422709/7a10df32-0dd9-4c3d-98f1-6dbbbdca0470


## Pre-merge Checklist
